webapp/src/app/shared/phaser/entities/entity-manager.ts

```diff
@@ -126,7 +128,7 @@ export class EntityManager extends BaseObject {
 						entity.startOffset.y = pointer.worldY - entity.container.y;
 						entity.isDragged = true;
 					});
-				} else {
+				} else if (!this.panKey.isDown){
```

Comment:
@Vegita2 The reason why I put this here, rather than including this in the return logic where at the begining of the function where `!pointer.leftButtonDown()` is to allow for the capability to multi select while panning.  Upon closer inspection, I do see the entities wiggle when panning,  so I would like to propose adding an `if panKey.isDown then return` statement after the entity has been selected and before the dragging